eng/emitter-package.json

```diff
@@ -9,7 +9,7 @@
     "@azure-tools/typespec-azure-resource-manager": "0.54.0",
     "@azure-tools/typespec-azure-rulesets": "0.54.0",
     "@azure-tools/typespec-client-generator-core": "0.54.0",
-    "@azure-tools/typespec-liftr-base": "0.7.0",
+    "@azure-tools/typespec-liftr-base": "0.8.0",
```

Comment:
We should not manually update this.
There will be an auto PR doing this when we upgrade it in autorest.csharp